muni_portal/tests/api/test_service_page.py

```diff
@@ -5,16 +5,20 @@
 from wagtail.images.models import Image
 from wagtail.images.tests.utils import get_test_image_file
 
-from muni_portal.core.models import ServicePage, AdministratorPage
+from muni_portal.core.models import (
+    ServicePage, AdministratorPage, ServicePointPage
+)
+
+OFFICE_HOURS_TEST_TEXT = "<div>Office hours text</div>"
 
 
 class ServicePageApiTestCase(TestCase):
 
-    def setUp(self):
-        self.client = Client()
-        self.url = reverse("wagtailapi:pages:listing")
+    @classmethod
+    def setUpTestData(cls):
+        cls.site = Site.objects.first()
 
-    def test_service_page_image(self):
+        # TODO: move this outside as common fixture ???
```

Comment:
instead of TODOs, let's create issues in the repo. Otherwise no one will see this when they should, and it will be in the way until it gets considered a priority.

Reply:
sure, will be doing this way in the future, but I guess it is not worth to create a new issue based on my this comment as it was just a suggestion unless you think it is worth take attention this right now